connect/mirror/src/test/java/org/apache/kafka/connect/mirror/TestUtils.java

```diff
@@ -32,4 +71,141 @@
         }
         return props;
     }
+    
+    public static void waitUntilMirrorMakerIsRunning(EmbeddedConnectCluster connectCluster, List<Class> mirrorClasses,
```

Comment:
This is the simple move from `MirrorConnectorsIntegrationTest` with generalization of connector class